src/app/client/src/app/modules/dashboard/components/report/report.component.html

```diff
@@ -257,25 +261,42 @@
                         </div>
                       </div>
 
-                      <div class="ui bottom attached segment p-0 b-0 no-bg my-24" suiTabContent="graph">
+                     <div class="report-block ui bottom attached segment p-0 b-0 no-bg my-24" suiTabContent="graph" >
+                      <div class="" *ngIf="currentReport?.reportMetaData?.filters && !hideElements">
+                       <p class="select-filter-label" > Select Filters <small> (This will apply for all the chart sections)</small> </p> 
```

Comment:
No hardcoded labels. Please put this label inside the translation file

Reply:
made changes